bookmarks/api/routes.py

```diff
@@ -94,7 +94,17 @@ def get_queryset(self):
     def get_serializer_context(self):
         return {'user': self.request.user}
 
+class UserViewSet(viewsets.GenericViewSet):
+    def get_serializer_context(self):
+        return {'user': self.request.user}
+    
+    @action(methods=['get'], detail=False)
+    def profile(self, request):
```

Comment:
Please add a basic test to `test_bookmarks_api.py`

Maybe set some profile properties to non-default values, get profile, verify values are correct.